.eleventy.js

```diff
@@ -129,7 +130,14 @@ module.exports = function (config) {
     html: true,
     breaks: true,
     linkify: false,
-  }).use(markdownItAttrs).use(markdownItAnchor);
+  })
+    .use(markdownItAnchor,{
+      permalink: markdownItAnchor.permalink.headerLink({ 
+        safariReaderFix: true,
```

Comment:
**Question:** Would we need we need this fix for any other browsers / what was the issue that we ran into for Safari's Reader mode? 😅

Reply:
I don't believe so! I think this is a temporary fix by the markdownItAnchor devs to address a Safari bug so I hopefully don't anticipate any other browser issues, but good point to keep a look at the library for any other accessibility issues they may find in the future